libc/include/time.yaml

```diff
@@ -37,6 +37,14 @@ functions:
     arguments:
       - type: const time_t *
       - type: char *
+  - name: ctime_s
+    standard:
+      - stdc
+    return_type: int
```

Comment:
`ctime_s` returns an `errno_t`; similar to what you've done for `rsize_t` (here and in the implementation itself).